Item Tracker Hookshot/Longshot Identifiers

Patrick12115 opened the pull request:
Adds an H and L for Hookshot and Longshot to be able to tell more clearly which one you have, since the icons are very similar.

![image](https://github.com/HarbourMasters/Shipwright/assets/115201185/c2c1876c-5e19-4df0-8db8-6371bfe3cbee)
![image](https://github.com/HarbourMasters/Shipwright/assets/115201185/7547999e-bf4e-499c-bda7-e49c9d7a9fc4)

https://github.com/HarbourMasters/Shipwright/assets/115201185/2f9e99a0-42ed-49fb-a7e8-2b4c3e5cb5c9



<!--- section:artifacts:start -->
### Build Artifacts
  - [soh.otr.zip](https://nightly.link/HarbourMasters/Shipwright/actions/artifacts/1207807622.zip)
  - [soh-linux-compatibility.zip](https://nightly.link/HarbourMasters/Shipwright/actions/artifacts/1207807623.zip)
  - [soh-linux-performance.zip](https://nightly.link/HarbourMasters/Shipwright/actions/artifacts/1207807624.zip)
  - [soh-mac.zip](https://nightly.link/HarbourMasters/Shipwright/actions/artifacts/1207807625.zip)
  - [soh-switch.zip](https://nightly.link/HarbourMasters/Shipwright/actions/artifacts/1207807626.zip)
  - [soh-wiiu.zip](https://nightly.link/HarbourMasters/Shipwright/actions/artifacts/1207807627.zip)
  - [soh-windows.zip](https://nightly.link/HarbourMasters/Shipwright/actions/artifacts/1207807628.zip)
<!--- section:artifacts:end -->

Patrick12115 committed:
H & L

Adds an H and L for Hookshot and Longshot to be able to tell more clearly which one you have.

Pepe20129 commented:
I think this should be an option as some people (specially those playing with mods that make the textures less similar) might prefer to not have this enabled.

Patrick12115 commented:
> I think this should be an option as some people (specially those playing with mods that make the textures less similar) might prefer to not have this enabled.

I can do that

Patrick12115 committed:
Added Toggle and Refined

Adds a toggle if players don't want it on. Also scales the letter to stay constant with the icons, both size and position. Also now hides the letter if you don't have a hookshot yet.

Patrick12115 commented:
> > I think this should be an option as some people (specially those playing with mods that make the textures less similar) might prefer to not have this enabled.
> 
> I can do that

Added a toggle if players don't want it on. Also now scales the letter to stay constant with the icons, both size and position and now hides the letter if you don't have a hookshot yet.

Patrick12115 committed:
Clean up some of the comments

PurpleHato commented:
Great addition! 
I just have a small suggestion regarding the placement of the text. I believe it would be more consistent if it followed the same logic as the ammo in terms of positioning. Currently, it's not at the same height, and if the option is toggled, you might also want to adjust its alignment to the left when necessary.

Patrick12115 commented:
> Great addition! 
> 
> I just have a small suggestion regarding the placement of the text. I believe it would be more consistent if it followed the same logic as the ammo in terms of positioning. Currently, it's not at the same height, and if the option is toggled, you might also want to adjust its alignment to the left when necessary.

Moving it to where it was was a personal choice for 2 reasons. One to follow how the other trackers out there do it, for familiarity, and two because the text grows with the icon and if it was below, it would start going into the icon below, which does not look good. Also no idea what you mean by move the alignment to the left.

Patrick12115 committed:
prefix
